lib/src/components/NxDropdown/NxDropdown.scss

```
@@ -37,9 +37,8 @@
   display: flex;
   margin: 0;
   outline: none;
+  padding: calc(var(--nx-spacing-2x) - 1px) calc(var(--nx-spacing-4x) - 1px);
```

```suggestion
  padding: calc(var(--nx-spacing-2x) - 1px) var(--nx-spacing-4x);
```
The left & right padding was already correct, so you should just have to override the top & bottom padding to always subtract 1px (instead of 2px in dark mode inherited from nxButton)

---

> The left & right padding was already correct

Is it though? Sketch shows 16px left and right _including the border_.  So by setting the CSS padding to 16px dropdowns are currently 2px too wide in both light and dark mode

---

The left and right padding was not correct, so it needs to be overridden here to just subtract 1px instead of 2px(button border width) in dark mode.

---

>>Is it though? Sketch shows 16px left and right including the border. So by setting the CSS padding to 16px dropdowns are currently 2px too wide in both light and dark mode

> I think the left and right padding was not correct, so it would need to be overridden here to just subtract 1px instead of 2px(button border width) in dark mode.

Yep you're right! I was looking at main and actually it's currently wrong then. It's currently set to 16px of padding plus 1 px of border, so this would resolve indiscrepencies in light mode as well.